docs/site-example/i18n/de.po

```diff
@@ -61,7 +61,7 @@ msgstr ""
 "Freifunk-Netz aufbauen, wenn andere Freifunk-Knoten in WLAN-Reichweite sind.</p>"
 "<p>Bitte schicke uns eine E-Mail mit dem Namen deines Knotens "
 "(<em><%= pcdata(hostname) %></em>) und ein paar Informationen an <a href="
-"\"mailto:freifunk-keys@lists.in-kiel.de?"
+"\"mailto:kontakt@alpha-centauri.freifunk.net?"
```

Comment:
```suggestion
"\"mailto:keys@alpha-centauri.freifunk.net?"
```

Reply:
This message block is `gluon-config-mode:novpn` (regular communication), as this is non-key communication, the kontakt-address should be used.